OneCommunityGlobal/HGNRest
# Shihao - Add 'members' to the Team schema

# Description
![Screenshot 2023-08-18 at 13 52 18](https://github.com/OneCommunityGlobal/HGNRest/assets/62367016/be665c8e-aa87-4aaf-b430-1c5db811661f)
To implement the sort by added date. It requires adding a new field to record it.

## Related PRS:
This PR is related to the [#1136](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1136) frontend PR.

## Main changes explained:
- Add `members` to the Team schema
- Create a script to add team documents to the database

## How to test:
1. check into current branch
2. ❗️ for the product environment only, run `node src/utilities/addMembersToTeams.js` to update the database schema (already update the dev database)
4. do `npm install` and `npm run build` to run this PR locally
5. start the server by `npm start`
6. ❗️ follow the instruction in the frontend PR [#1136](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1136)

## Screenshots or videos of changes:
**Instruction to run the script**

https://github.com/OneCommunityGlobal/HGNRest/assets/62367016/89c9396e-be0b-41a8-ba3b-518899c11b08
